src/utils/useAuth.tsx

```diff
@@ -0,0 +1,5 @@
+import { getLocalStorage } from "@/utils";
+
+export const useAuth = () => {
+  return getLocalStorage("user");
```

Comment:
same comment as in previous PR

Reply:
I agree, once I merge I'll keep the changes made in the other PR